host/class/cdc/usb_host_cdc_acm/test_app/main/test_cdc_acm_host.c

```diff
@@ -599,6 +599,40 @@ TEST_CASE("closing", "[cdc_acm]")
     vTaskDelay(20); //Short delay to allow task to be cleaned up
 }
 
+/* Basic test to check CDC driver reaction to TX timeout */
+TEST_CASE("tx_timeout", "[cdc_acm]")
+{
+    nb_of_responses = 0;
```

Comment:
We are not checking this value anywhere in this test case. Can we remove it, not to confuse anyone?
Same comment from the test case above, probably a copy-paste of the test case.